lib/offsite_payments/integrations/coinbase.rb

```diff
@@ -114,7 +113,7 @@ def acknowledge(authcode = {})
           posted_order = @params
           parse(response)
 
-          %w(id custom total_native status).all? { |param| posted_order[param] == @params[param] }
+          %w(id custom total_native status).all? { |param| posted_order[param] == @params[param] } unless @params.nil?
```

Comment:
I think this method normally returns `true/false`. This change will make it return `nil`. Should we instead explicitly return `false` if `@params` is `nil`?
```
return false unless @params
```